sql_queries/Populate_SpatialSubsetTable_csiro_cmar_trajectory.sql

```diff
@@ -0,0 +1,41 @@
+SET SEARCH_PATH = marvl3, public;
+-- CSIRO TRAJECTORY
+\echo 'CSIRO TRAJECTORY'				
+INSERT INTO marvl3.spatial_subset(
+source_id,
+origin_id,
+"LONGITUDE",
+"LONGITUDE_QC",
+"LATITUDE",
+"LATITUDE_QC",
+"TIME",
+"TIME_QC",
+"DEPTH",
+"DEPTH_QC",
+"TEMP",
+"TEMP_QC",
+"PSAL",
+"PSAL_QC",
+geom
+)
+SELECT 
+s.source_id,
+d."SURVEY_ID",
+d."LONGITUDE",
+'0',
+d."LATITUDE",
+'0',
+d."TIME" AT TIME ZONE 'UTC' AS TIME,
+'0',
+gsw_z_from_p(d."PRESSURE", d."LATITUDE")  -- looks like pressure is  PRES_REL , value close to 0dbar near surface
```

Comment:
gsw_z_from_p provides depth positive up, so need to add "-" to change direction (positive down).
I do realise this is a bit dodgy but this is the original gibbs sea water convention and I thought I'd be good to stick with it since I ported the code without modification.